src/Pages/Home/Home.jsx

```diff
@@ -0,0 +1,9 @@
+import Navbar from "../../components/Navbar/Navbar";
```

Comment:
better create an ```index.js```  file in the components folder, and make all the imports and exports from there to keep the codebase clean